### ISSUE TITLE
Does the API differ from official Javascript API?

### ISSUE THREAD

**Issue by lulalala** [vivekmarakana/iota.lib.rb#7]
I tried to call `getNewAddress` like how it's done in the javascript library:

```
iota.api.getNewAddress(seed, {index: 0 , total: 1, security: 3, checksum: true})
```

and I get a `NoMethodError: undefined method `getNewAddress'`

The method is part of the Account class, so it has to be accessed differently.

So is this a faithful port or will take the strength of Ruby's OO architecture? And can we clarify that in the readme?

**Comment by nofxx** [comment]
Account#getNewAddress works, but you need to be connected to a node.
So I'm using Seed#getAddress. Notice, not named parameters: eg ```getAddress(0, 2, false)```

**Comment by lulalala** [comment]
Yep I am using it that way. I guess what I am trying to say is, the readme should be updated, because this is not a 1:1 port of JS library.

```
require 'iota'
iota = IOTA::Client.new(...)
iota.api.getNewAddress # NoMethodError
```
